docs/API.md

```diff
@@ -15,6 +15,8 @@
 
 * [Handler Components](#handler-components)
 
+* [Named Components](#named-components)
```

Comment:
This is actually a level 3 heading under `Handler Components` so it should be nested under that.